io.github.ja2-stracciatella.yaml

```diff
@@ -0,0 +1,91 @@
+app-id: io.github.ja2-stracciatella
+runtime: org.freedesktop.Platform
+runtime-version: '22.08'
+sdk: org.freedesktop.Sdk
+sdk-extensions:
+  - "org.freedesktop.Sdk.Extension.rust-stable"
+command: ja2-launcher-wrapper
+rename-desktop-file: ja2-stracciatella.desktop
+rename-icon: ja2-stracciatella
+finish-args:
+  - '--share=network'
+  - '--socket=x11'
+  - '--socket=wayland'
```

Comment:
you said it doesn't support waylad out of the box, so this should be removed.

```suggestion
```

Reply:
I said you'd need to set a flag in the environment.  If a user sets `SDL_VIDEODRIVER=wayland` anywhere in their profile it propagates to the flatpak environment, and without access to the wayland socket the game would just silently fail to start.

Reply:
again, not out of the box.

Reply:
So, if the user sets that flag because they prefer wayland, and JA2 then just silently stops working… that's okay?

And in any case, [App Requirements](https://github.com/flathub/flathub/wiki/App-Requirements#unrestricted-permissions) says that `--socket=wayland` can be freely used, so why do I have to remove it?